jena-rdfconnection/src/main/java/org/apache/jena/rdflink/RDFLinkHTTP.java

```diff
@@ -221,163 +220,54 @@ public Graph queryDescribe(String queryString) {
     public boolean queryAsk(String queryString) {
         return
             Txn.calculateRead(this, ()->{
-                try ( QueryExec qExec = query(queryString, QueryType.ASK) ) {
+                try ( QueryExec qExec = query(queryString) ) {
                     return qExec.ask();
                 }
             } );
     }
 
-    /**
```

Comment:
Could you say which this change is in the PR? The `QueryType` may no tbe partiucaly useful in this impl but when inheriited it might be useful.